tests/fpgaperf/fpgaperf_counter.h

```diff
@@ -0,0 +1,153 @@
+// Copyright(c) 2021, Intel Corporation
```

Comment:
This header seems to be duplicated.  Instead of doing that, we should add the necessary include directory when building the tests.